op-e2e/e2eutils/disputegame/helper.go

```diff
@@ -178,8 +178,8 @@ func (h *FactoryHelper) StartOutputCannonGame(ctx context.Context, l2Node string
 	h.require.NoError(err, "create fault dispute game")
 	rcpt, err := wait.ForReceiptOK(ctx, h.client, tx.Hash())
 	h.require.NoError(err, "wait for create fault dispute game receipt to be OK")
-	h.require.Len(rcpt.Logs, 1, "should have emitted a single DisputeGameCreated event")
-	createdEvent, err := h.factory.ParseDisputeGameCreated(*rcpt.Logs[0])
+	h.require.Len(rcpt.Logs, 2, "should have emitted a single DisputeGameCreated event")
```

Comment:
nit
```suggestion
	h.require.Len(rcpt.Logs, 2, "should have emitted a DisputeGameCreated event")
```

Reply:
Ah it's still a single DisputeGameCreated event but now it also emits a Withdrawal event